src/datastore/collections.h

```diff
@@ -31,9 +33,11 @@ class Collection {
 	void               name(const std::string &name) noexcept { _data.name = name; }
 	void               name(std::string &&name) noexcept { _data.name = std::move(name); }
 
-	const members_t members() const;
-	void            add(const member_t &id) const;
-	void            remove(const member_t &id) const;
+	const AccessPolicies accessPolicies() const;
```

Comment:
Ideally these should be moved to `access-policies.h` (and `rbac-policies.h`). e.g.  `const AccessPolicies RetrieveAccessPoliciesByCollection(const std::string &id);`. 

The reasoning here is `Collection` should be able to operate independently of access policies.